tsdate/core.py

```diff
@@ -954,32 +955,37 @@ class ExpectationPropagation(InOutAlgorithms):
     Bayesian Inference"
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, global_prior, **kwargs):
```

Comment:
I'm not sure I quite understand how the global prior is used differently from self.prior here. Is it worth adding a docstring to explain?